types/adobe__pdfservices-node-sdk/operation/replace-pages-operation.d.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * An operation that allows specific pages in a PDF file to be replaced with pages from multiple PDF files.
+ * <p>
+ * For more complex use cases, refer the {@link CombineFilesOperation}
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        ReplacePages = PDFServicesSdk.ReplacePages,
+ *        replacePagesOperation = ReplacePages.Operation.createNew();
+ *
+ *  replacePagesOperation.setBaseInput(PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/replacePagesOperationBaseInput.pdf',
+ *  	ReplacePages.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/replace-pages-operation.d.ts;
  Line: 17;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.